src/PulseFinding.cpp

```diff
@@ -25,8 +25,10 @@ void simple_threshold_technique(TH1D *hwaveform, WaveformFitResult *fitresult, P
   // Loop over waveform; look for every case of waveform going below fixed threshold
   
   double baseline = 1.0;
-  if(pmt.channel == 0){ baseline = 0.991; }
-  if(pmt.channel == 1){ baseline = 0.9966; }
+  if(pmt.channel == 0){ baseline = 0.9915; }
```

Comment:
I see this is spanning more than one file now, so would need a common header/configuration file (for the comment above).

Reply:
I do want to clean this up; my suggestion was to use the baselines that were calculated in the online analysis as the baseline for each channel.  Yes, this change does need to be made in several places.

I could make that change as part of this PR, but it is really a different bit of work, pulling in the baselines from the online analysis and figuring out how to propagate them through the code.  Seems better to be separate.